apps/dotcom/sync-worker/src/TLDrawDurableObject.ts

```diff
@@ -298,6 +325,29 @@ export class TLDrawDurableObject {
 		const { 0: clientWebSocket, 1: serverWebSocket } = new WebSocketPair()
 		serverWebSocket.accept()
 
+		const closeSocket = (code: number, message: string) => {
+			serverWebSocket.close(code, message)
+			return new Response(null, { status: 101, webSocket: clientWebSocket })
```

Comment:
i see this pattern is in other code files but it's surprising to me that after the `close` we send a 101, is that the correct status code for this situation?

Reply:
it might still work with other status codes? you can try it. but http status codes aren't exposed on the client for websocket requests, for some security reason. that's why we pass a code to the close method instead. so it's not helpful beyond making this code look more like normal http code